editor/editor_themes.cpp

```diff
@@ -241,7 +241,7 @@ void editor_register_and_generate_icons(Ref<Theme> p_theme, bool p_dark_theme =
 	dark_icon_color_dictionary[Color::html("#45ff8b")] = success_color;
 	dark_icon_color_dictionary[Color::html("#dbab09")] = warning_color;
 
-	ImageLoaderSVG::set_convert_colors(&dark_icon_color_dictionary);
+	// ImageLoaderSVG::set_convert_colors(&dark_icon_color_dictionary);
```

Comment:
This might need to be re-implemented? I haven't checked but I guess it's used to convert colors from dark theme to light theme?

Reply:
We discussed in a thread with @Calinou that the SVG importer should not be replacing colors..

That the operation should be done on the xml file. Stuck here.